content/en/docs/instrumentation/php/automatic.md

```diff
@@ -79,7 +79,43 @@ function will record it, without affecting exception propagation.
 The extension can be installed via pecl,
 [pickle](https://github.com/FriendsOfPHP/pickle) or
 [php-extension-installer](https://github.com/mlocati/docker-php-extension-installer)
-(docker specific).
+(docker specific). There are also packaged versions of the extension available
+for some Linux package managers.
+
+### Linux packages
+
+RPM and APK packages are provided by the following:
+
+- [Remi repository](https://blog.remirepo.net/pages/PECL-extensions-RPM-status) -
+  RPM
+- [Alpine linux](https://pkgs.alpinelinux.org/packages?name=*pecl-opentelemetry) -
+  APK (currently in the
+  [_testing_ branch](https://wiki.alpinelinux.org/wiki/Repositories#Testing))
+
+{{< tabpane text=true >}} {{% tab "RPM" %}}
+
+```sh
+#this example is from centos 7
+yum update -y
+yum install -y epel-release yum-utils
+yum install -y http://rpms.remirepo.net/enterprise/remi-release-7.rpm
+yum-config-manager --enable remi-php82
+yum install -y php php-pecl-opentelemetry
+
+php --ri opentelemetry
+```
+
+{{% /tab %}} {{% tab "APK" %}}
+
+```sh
+echo "@testing https://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories
+apk add php php81-pecl-opentelemetry@testing
```

Comment:
is there a reason why this is for PHP 8.1 and the RPM for PHP 8.2? Do both exist? Might it be worthwhile to add a comment that people should make sure here to pick the right version for the PHP they use?

Reply:
They do both exist, I'll add a note about it.

Reply:
thanks